admin/users/views.py

```diff
@@ -0,0 +1,40 @@
+from django.shortcuts import redirect
+
+from website.project.model import User
+
+from admin.base.views import GuidFormView, GuidView
+from admin.users.templatetags.user_extras import reverse_user
+from .serializers import serialize_user
+
+
+def disable_user(request, guid):
+    user = User.load(guid)
+    # user.disable_account()
+    user.is_disabled = True  # TODO: change to user.disable_account() after local tests
+    user.save()
+    return redirect(reverse_user(guid))
+
+
+def reactivate_user(request, guid):
```

Comment:
This method also needs to resubscribe the user to `website.settings.MAILCHIMP_GENERAL_LIST` if `disable_user` is unsubscribing them (which it seems like it should). See https://github.com/CenterForOpenScience/osf.io/blob/develop/website/mailchimp_utils.py#L98-L101